source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -424,15 +451,15 @@ export default class WalletRestoreDialog extends Component<Props, State> {
           skin={InputSkin}
         />
 
-        {!this.isCertificate() && (
+        {!this.isCertificate() && !this.isYoroi() && (
```

Comment:
@DeeJayElly please replace:
`!this.isCertificate() && !this.isYoroi()`
with:
`this.isRegular() && this.isLegacy()`

Reply:
Fixed